workbench-core/accounts/src/services/projectService.ts

```diff
@@ -307,33 +312,26 @@ export default class ProjectService {
     // verify project exists
     await this.getProject({ projectId: request.projectId });
 
-    // Delete Permissions for the groups--TODO implement after dynamic AuthZ
-    // const identityPermissions: IdentityPermission[] = this._generateIdentityPermissionsForProject(projectId);
-    // const deleteIdentityPermissionsResponse = this._dynamicAuthorizationService.deleteIdentityPermissions(
-    //   identityPermissions
-    // );
-    // if (!createIdentityPermissionsResponse.deleted) {
-    //   throw Boom.badImplementation(
-    //     'Failed to delete batch identity permissions for project with dyamic authorization service.'
-    //   );
-    // }
-
-    // Delete ProjectAdmin and Researcher groups--TODO implement after dynamic AuthZ
-    // const deleteProjectAdminGroupResponse = this._dynamicAuthorizationService.deleteGroup(user, {
-    //   groupId: `${projId}#PA`
-    // });
-    // const deleteResearcherGroupResponse = this._dynamicAuthorizationService.deleteGroup(user, {
-    //   groupId: `${projId}#Researcher`
-    // });
-    // if (!deleteProjectAdminGroupResponse.created || !deleteResearcherGroupResponse.created) {
-    //   throw Boom.badImplementation(
-    //     'Failed to delete Project Admin group or Researcher group with dynamic authorization service.'
-    //   );
-    // }
+    const identityPermissions: IdentityPermission[] = this._generateIdentityPermissionsForProject(
+      request.projectId
```

Comment:
nit: we could probably extract attributes of `request` out at the beginning of the file since they are used multiple times